Manual/manJHUGenerator.tex

```diff
@@ -796,12 +796,10 @@ \subsubsection{Couplings}
 complex(8), public :: ghg2 = (1.0d0,0d0)
 complex(8), public :: ghg3 = (0.0d0,0d0)
 complex(8), public :: ghg4 = (0.0d0,0d0)   ! pseudoscalar
```

Comment:
Would it be possible to put the whitespaces in the manual back?

Reply:
Maybe? Do you mean just convert tabs to spaces again?

Reply:
No, what I meant was to not make whitespace changes in the manual because it affects LaTeX formatting. If you compile it, it would affect how verbatim statements appear, or how paragraphs are split.

Reply:
Yeah I see what you are saying now, I will try my best to fix it right away. I'm unsure why is changed the white spaces since I literally just pulled the tex from master and added my section.

Reply:
Should now be consistent with master